platform/nservicebus.include.md

```diff
@@ -1,5 +1,68 @@
-NServiceBus, the heart of the system, is a messaging and workflow framework. It helps create distributed systems that are scalable, reliable, and easy to change. It supports various messaging patterns, encapsulates long-running business processes as [sagas](/nservicebus/sagas), and provides abstractions over several [queuing](/transports/) and [storage](/persistence/) technologies.
+NServiceBus is the heart of a distributed system and the Particular Service Platform. It helps create systems that are scalable, reliable, and flexible.
 
-Most queuing technologies try to guarantee 'at least once', or even 'exactly once', message delivery. They usually fall short of this promise. NServiceBus solves intermittent delivery problems by  automatically retrying messages using various strategies. If all else fails, it forwards messages to an error queue. From there, [ServiceControl](/servicecontrol/) and [ServicePulse](/serviceinsight/) expose the problems for human intervention.
+At its core, NServiceBus works by routing _messages_ between _endpoints_. [Messages](/nservicebus/concept-overview.md#message) are plain C# classes that contain meaningful data for the business process that is being modeled.
```

Comment:
Do we need to provide a link to explain what is endpoint?

Reply:
There is one at the top of the next paragraph. The idea was to define a message, show one, then define an endpoint, and show one.